src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/ErrorConsumerImpl.java

```diff
@@ -59,7 +66,12 @@ public void readAndProcessErrorTopic(@Payload ChipsRestInterfacesSend data,
         logger.infoContext(messageId, String.format("%s: Consumed Message from Partition: %s, Offset: %s", groupId, partition, offset), logMap);
         logger.infoContext(messageId, String.format("received data='%s'", data), logMap);
 
-        messageProcessorService.processMessage("error-consumer", data);
+        List<String> failedMessageIds = new ArrayList<>();
```

Comment:
I think all the changes to this class (ErrorConsumerImpl) can be reverted now, just pass an extra null to the messageProcessor. When we call the messageProcessor, as it is running in error mode for this class, it will place message on retry topic and return and won't update the arraylist we were passing in
this is the section in the messageProcessor that will just return - 
![image](https://user-images.githubusercontent.com/11330756/109515889-81bd7d00-7a9f-11eb-94bd-6ad44d40c2a5.png)